examples/pylintrc

```diff
@@ -93,7 +93,7 @@ prefer-stubs=no
 
 # Minimum Python version to use for version dependent checks. Will default to
 # the version used to run pylint.
-py-version=3.13
+py-version=3.12
```

Comment:
Would have expected this to be handled correctly by:

https://github.com/pylint-dev/pylint/blob/42aa204b3d27a712588c3800b017149c74947486/doc/exts/pylint_options.py#L43

Ho well.